mockito-kotlin/build.gradle

```diff
@@ -25,33 +26,34 @@ dependencies {
     compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     compileOnly 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.0'
 
-    compile "org.mockito:mockito-core:5.3.1"
+    implementation "org.mockito:mockito-core:5.7.0"
 
-    testCompile 'junit:junit:4.13.2'
-    testCompile 'com.nhaarman:expect.kt:1.0.1'
+    testImplementation 'junit:junit:4.13.2'
+    testImplementation 'com.nhaarman:expect.kt:1.0.1'
 
-    testCompile  "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    testCompile  "org.jetbrains.kotlin:kotlin-test:$kotlin_version"
-    testCompile 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.0'
+    testImplementation  "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    testImplementation  "org.jetbrains.kotlin:kotlin-test:$kotlin_version"
+    testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.0'
 
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.0.0"
 }
 
-dokka {
-    outputFormat = 'html'
-    outputDirectory = "$buildDir/javadoc"
+dokkaHtml.configure {
+    outputDirectory.set(file("$buildDir/javadoc"))
 
-    linkMapping {
-        dir = "src/main/kotlin"
-        url = "https://github.com/nhaarman/mockito-kotlin/tree/master/mockito-kotlin/src/main/kotlin"
-        suffix = "#L"
+    dokkaSourceSets {
+        named("main") {
+            localDirectory.set(file("src/main/kotlin"))
+            remoteUrl.set(new URL("https://github.com/nhaarman/mockito-kotlin/tree/master/mockito-kotlin/src/main/kotlin"))
+            remoteLineSuffix.set("#L")
+        }
     }
 }
 
 tasks.withType(KotlinCompile).configureEach {
-    kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_11
+    compilerOptions {
+        jvmTarget.set(JvmTarget.JVM_17)
```

Comment:
Mockito still supports Java 11, so we can't bump this here. Please revert the changes to Java 17, the rest looks good to me.